heart-rate-sensor/heartRateSensor.js

```diff
@@ -95,7 +96,11 @@
     _readCharacteristicValue(characteristicUuid) {
       let characteristic = this._characteristics.get(characteristicUuid);
       return characteristic.readValue()
-      .then(buffer => new DataView(buffer));
+      .then(value => {
+        // In Chrome 50+, a DataView is returned instead of an ArrayBuffer.
+        value = value.buffer ? value : new DataView(value);
```

Comment:
I'd just write `return value.buffer ? value : new DataView(value);`, although it doesn't matter much.